run_agent.sh

```diff
@@ -60,6 +60,17 @@ if [ "$AGENT_ENVIRONMENT" = "preview" ] && [ -n "$PR_NUMBER" ]; then
         cd "$AGENT_DIR"
         cp ../.env .
         poetry install
+        
+        if [ -d "browser" ]; then
+            echo "Setting up browser service..."
+            cd browser
+            npm install
+            npm run build
```

Comment:
Would love to achieve these steps in ci, and then only need to run npm start on the compiled artifacts here